### PR TITLE
[FIX] Changed Alignment 

### LINKED ISSUE #68: [FEATURE] - Alignment
### Describe the improvement

- [ ] Make the text equally padded from both sides.
- [ ] Review alignment and adjust to match the image below.

Reference image

### Why do we need this?

The requested improvement is essential for the following reasons:

1. **Aesthetic Consistency:** Equally padding text from both sides helps maintain a visually pleasing and consistent appearance in your content. It prevents text from looking cluttered or misaligned, ensuring a more professional and polished look.

2. **Readability:** Properly aligned content is easier to read and understand. Inconsistent alignment can cause confusion and make content less accessible, particularly when presenting information to a wide audience.

3. **Enhanced User Experience:** Users expect a certain level of alignment and padding in documents and web content. Meeting these expectations can enhance the overall user experience and satisfaction with your materials.

4. **Professionalism:** A well-organized and aligned document or design conveys professionalism and attention to detail. This can be particularly important in business, academic, or any context where a polished appearance is valued.

By addressing these aspects, the proposed improvement not only improves the aesthetics but also contributes to the overall quality and effectiveness of the content.

### Do you have a proposed solution?

We welcome any suggestions or ideas from external contributors on how to achieve the desired alignment and text padding. Your input and collaboration are valuable in finding the most suitable solution for this improvement.

Please feel free to share your insights, recommendations, or proposed solutions for this improvement.

Currently :
image

### Do you want to work on this feature?

- [ ] I am willing to implement this feature.

### Terms and Conditions

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Harshil-08**
## Title :

Changed Alignment of text in Certified section by adding padding.

## Issue no : 

This PR fixes #68 

## Description

given padding to the text in certified section.

## Screenshots (if applicable)

image

## Checklist
- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/tcet-opensource/.github/blob/main/CODE_OF_CONDUCT.md)
- [x] I have read and followed the **Contributing Guidelines**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -555,6 +555,11 @@
     "@jridgewell/resolve-uri" "3.1.0"
     "@jridgewell/sourcemap-codec" "1.4.14"
 
+"@kurkle/color@^0.3.0":
```
Comment (Horror26) [inline yarn.lock:558]: this package is not necessary delete it

Comment (Horror26) [reply yarn.lock:558]: there should not be any changes in yarn.lock

**Review by Horror26 — CHANGES_REQUESTED** [review]
there should not be any changes in yarn.lock other than that LGTM

**Comment by Harshil-08** [comment]
> there should not be any changes in yarn.lock other than that LGTM

Done

**Review by Horror26 — APPROVED** [review]
LGTM

**Pull request merged**